### ISSUE TITLE
Inference using darknet.py on video does not use GPU

### ISSUE THREAD

**Issue by SamScib** [AlexeyAB/darknet#7405]
I am writing a wrapper for Darknet YoloV4 and am having an issue running it on GPU (both debug and through console cmd 'python main.py' work,but have very low fps)

I tried to run 'python darknet_video.py' but it is very slow

I use 'self.set_gpu(0) # Set GPU' to make sure I use GPU with ID 0 (my GPU) after loading the yolo_cpp_dll.dll functions for python

when I check if CUDA is detected with PyTorch, it says it is and am able to use it with other frameworks, so I do not think it is a setup issue

Running the compiled darknet.exe does use GPU (25 fps), only python version is having issues.

Below are screenshots of parts where I am loading the network

![image](https://user-images.githubusercontent.com/49656532/108602388-f0188600-73a1-11eb-8730-021e20ebf063.png)

![image](https://user-images.githubusercontent.com/49656532/108602447-45549780-73a2-11eb-8323-173e35585403.png)

**Comment by haviduck** [comment]
is the dll in that path? and might need opencv stuff in your paths

**Comment by SamScib** [comment]
> is the dll in that path? and might need opencv stuff in your paths

Yes all DLLs in path, installed openCV from source and added to path as well

**Comment by haviduck** [comment]
> > is the dll in that path? and might need opencv stuff in your paths
> 
> Yes all DLLs in path, installed openCV from source and added to path as well

whats the arch? 
i have to compile opencv for 86. but maybe try a barebone script to ensure that everything works in py before creating custom. dumpbin /DEPENDENTS on yolo cpp dlls and darknet dll and see if all of its dependents are available.

**Comment by RohitSingh1226** [comment]
Will I get better performance When I use python script

**Comment by RohitSingh1226** [comment]
Did you find a solution? 
Also are you doing this on linux or Windows
